pkg/procmatch/procmatch_test.go

```diff
@@ -0,0 +1,124 @@
+package procmatch
+
+import (
+	"testing"
+)
+
+var (
+	rawIntegs = []Integration{
+		{"java", []string{"java"}},
+		{"elasticsearch", []string{"java org.elasticsearch.bootstrap.Elasticsearch"}},
+		{"kafka", []string{"java kafka.kafka"}},
+		{"gunicorn", []string{"gunicorn: master "}},
+		{"haproxy", []string{"haproxy", "haproxy-master"}},
+		{"mongodb", []string{"mongod"}},
+		{"consul", []string{"consul agent", "consul_agent", "consul-agent"}},
+		{"supervisord", []string{"python supervisord", "supervisord"}},
+		{"pgbouncer", []string{"pgbouncer"}},
+		{"apache", []string{"httpd", "apache"}},
+	}
+
+	rawIntegs2 = []Integration{
+		{"cassandra", []string{"java org.apache.cassandra.service.CassandraDaemon"}},
+		{"ceph", []string{"ceph-*"}},
+		{"couchbase", []string{"beam.smp"}},
+		{"couchdb", []string{"couchjs"}},
+		{"etcd", []string{"etcd"}},
+		{"fluentd", []string{"td-agent", "fluentd", "ruby td-agent"}},
+		{"kyototycoon", []string{"ktserver"}},
+		{"lighttpd", []string{"lighttpd"}},
+		{"memcached", []string{"memcached"}},
+		{"mysql", []string{"mysqld"}},
+	}
+
+	rawIntegs3 = []Integration{
```

Comment:
If we're only splitting these up for the benchmarks I think I'd lean towards keeping a single list and just grabbing pieces of the slice. This will make it clearer for the next person adding integrations with tests. I'd also recommend sorting all the integrations.